ROADMAP.md

```diff
@@ -0,0 +1,37 @@
+
+## Current Roadmap Projects
+
+| Project | Description | ETD | PR / Issue |
+|---------|-------------|-----|--------------------------|
+|Cucumber Framework Support|A lot of people request support for Cucumber as it is their main framework choice. Initial work on this has already been done by the community. The code needs to get proper unit tests but would then be good to go.|Q2|https://github.com/webdriverio/webdriverio/pull/3667|
+|Make CLI tool more powerful|Adding simple add ons to the setup becomes difficult if the person is not familiar with the project or WebDriver in general. Adding some simple commands to the CLI interface that allows to add service and reporters and modifies the config would make the process of adding plugins much easier.|Q2|https://github.com/webdriverio/webdriverio/issues/3915|
+
+## Upcoming Projects (in rough order of priority)
+
+| Project | Description |
+|---------|-------------|
+|TBD|TBD|
+
+## Projects Under Consideration (unplanned and unprioritized)
+
+| Project | Description |
+|---------|-------------|
+|Jest Framework Support|Jest has become one of the most popular unit test framework in the JS ecosystem. Even though most of their features are not helpful in the e2e space there are some things (e.g. snapshot testing) that could help people writing better e2e tests. There has been some initial work done on this.|
+|Auto Driver Setup|One of the most challenging problems when working with WebDriver is the setup of the driver. Even though WebdriverIO provides services around this that help to get up and running easier, it is still difficult to set up for someone who has no idea about how everything is connected. To simplify this it would be desire-able if WebdriverIO handles downloading and running drivers for all environments.|
+|Better Debugging Capabilities|There are already a [handful options](https://webdriver.io/docs/debugging.html) to debug test code with WebdriverIO. However it is still not straight forward to use native Node.js debugging capabilities which requires special handling of workers and sub processes. Goal has to be that developers can use their IDEs to set breakpoints to debug.|
```

Comment:
Isn't it possible to debug tests and webdriverIO itself right now? I can set breakpoints in my VSCode